### DISCUSSION TITLE
why there is no Accepted/Shipped helper ?

### DISCUSSION THREAD

**Discussion by TheElegantCoding** [solidjs-community/solid-primitives#536] [Q&A] [answered]
hi, I want to know why in all the function hooks and more no one is still Accepted/Shipped, what means that?

**Comment by thetarnav** [comment, marked as answer]
We are yet to ship a package where we combine and reexport all other packages.
Getting included in that package would be a good reason for utilizing the final stage.
Other then that I can’t think of any other purpose, it’s been mostly a way to judge the quality of the package.
